.github/workflows/dev_on_workflow_sentry_on_deploy.yml

```
@@ -0,0 +1,87 @@
+name: Install runtime environment and dependencies
+
+on:
+  workflow_call:
+    inputs:
+      ENV:
+        type: string
+        required: true
+    secrets:
+      SENTRY_AUTH_TOKEN:
+        required: true
+      ANDROID_APPCENTER_API_TOKEN:
+        required: true
+      IOS_APPCENTER_API_TOKEN:
+        required: true
+      SLACK_BOT_TOKEN:
+        required: true
+
+jobs:
+  sentry_and_deploy:
+    runs-on: [self-hosted, linux, X64]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Setup yarn
+        run: npm install -g yarn
+      - uses: actions/cache@v3
+        id: yarn-modules-cache
+        with:
+          path: |
+            node_modules
+            ~/.cache/yarn
+          key: v1-yarn-dependency-cache-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            v1-yarn-dependency-cache-${{ hashFiles('**/yarn.lock') }}
+      - name: 'Setup sentry credentials'
+        run: |
+          echo "[defaults]" >> ~/.sentryclirc
+          echo "url=https://sentry.passculture.team/" >> ~/.sentryclirc
+          echo "org=sentry" >> ~/.sentryclirc
+          echo "project=application-native" >> ~/.sentryclirc
+          echo "" >> ~/.sentryclirc
+          echo "[auth]" >> ~/.sentryclirc
+          echo "token=${{ secrets.SENTRY_AUTH_TOKEN }}" >> ~/.sentryclirc
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          #  - name: Deploy android for ${{ inputs.ENV }}
+          #    run: |
+          #      bundle exec fastlane android deploy codepush: --env ${{ inputs.ENV }}
+          #    env:
+          #      ANDROID_APPCENTER_API_TOKEN: ${{ secrets.ANDROID_APPCENTER_API_TOKEN }}
+          #  - name: Deploy ios App for ${{ inputs.ENV }}
+          #    run: |
+          #      bundle exec fastlane ios deploy codepush: --env ${{ inputs.ENV }}
+          #    env:
+          #      IOS_APPCENTER_API_TOKEN: ${{ secrets.IOS_APPCENTER_API_TOKEN }}
+      - uses: technote-space/workflow-conclusion-action@v3
+        if: ${{ always() }}
+      - name: Post to a Slack channel
+        id: slack
+        if: ${{ env.WORKFLOW_CONCLUSION == 'failure' }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
```

```suggestion
        with:
        # nom du channel slack
```
`nom` du channel slack à remplacer par la vrai valeur